utility/pkgdb2.cfg.sample

```diff
@@ -19,6 +19,9 @@ DB_URL='sqlite:////var/tmp/pkgdb2_dev.sqlite'
 ### pages
 ITEMS_PER_PAGE = 50
 
+### Blacklisted items
```

Comment:
But maybe we could include it in the doc, something like:
~~~~
### Blacklisted items (following the format: ``namespage/packagename``)
~~~~
Thoughts?